### PR TITLE
Adding Mailhog as a transport, for local development.

### PR DESCRIPTION
**Pull request by tineke-corin**

### CODE REVIEW

File: lib/transports/mailhog.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+var nodemailer = require('nodemailer');
+var debug = require('debug')('loopback-component-mailer');
+
+module.exports = function (options) {
+    const transporter = nodemailer.createTransport({
+        host: 'localhost',
+        port: 1025,
+    });
```
Comment (markstuart) [inline lib/transports/mailhog.js:8]: Allowing these to come in from the options might be nice:
```js
const defaults = {
  host: 'localhost',
  port: 1025
}
const transportOptions = Object.assign({}, defaults, options)
const transporter = nodemailer.createTransport(transportOptions)
```
Then the consumer can set their own host and port values as needed

File: package.json [hunk 2]
```diff
@@ -18,14 +18,15 @@
...
+    "@sendgrid/mail": "^6.3.1",
     "debug": "^2.2.0",
     "deepmerge": "^0.2.10",
     "handlebars": "^4.0.4",
     "kue": "^0.11.0",
-    "strong-error-handler": "^3.2.0",
     "loopback": "^3.22.2",
-    "@sendgrid/mail": "^6.3.1",
-    "striptags": "^2.0.4"
+    "mailhog": "^4.16.0",
```
Comment (markstuart) [inline package.json]: Not sure why this would be needed as a dependency?

Comment (tineke-corin) [reply package.json]: Quite right, I thought I wanted it but changed my mind 🙄

File: lib/transports/mailhog.js (hunk 1 above)
Comment (tineke-corin) [reply lib/transports/mailhog.js:8]: Yep, nice idea - I have added this

File: README.md [hunk 3]
```diff
@@ -48,13 +48,34 @@ Options:
...
 
+  The apiKey is required for sendgrid only.
+
+  You can use Mailhog to capture and display emails in local development.  The simplest way is to spin up Mailhog
+  in a docker container:
+
+  `docker run -p 8025:8025 -p 1025:1025 mailhog/mailhog`
+
+  You can then browse to localhost:8025 to see the emails that have been sent. If you run mailhog on non-default
+  host or port (localhost:1025) then add the host and port into the configuration in your component-config.json:
```
Comment (markstuart) [inline README.md:66]: Nice doc update :+1:

**Review by markstuart — APPROVED** [review]
:shipit:

**Pull request merged**